tool/src/main/java/org/datacommons/tool/Processor.java

```diff
@@ -105,8 +110,9 @@ public static Integer process(Args args) throws IOException {
         processor.processNodes(Mcf.McfType.TEMPLATE_MCF);
       }
 
+      // We've been adding stats to statChecker all along, now do the actual check.
       if (args.doStatChecks) {
```

Comment:
nit: be consistent with whether statChecker check happens outside (here) or inside (addStats) ?

Reply:
also, perhaps checkStats() can just inlined

Reply:
done